tests/BenchmarkDotNet.Tests/TestCultureInfo.cs

```diff
@@ -0,0 +1,15 @@
+using System.Globalization;
+using BenchmarkDotNet.Helpers;
+
+namespace BenchmarkDotNet.Tests
+{
+    internal static class TestCultureInfo
```

Comment:
same as above, do we really need this extra type?

Reply:
Yes, it will simplify mocking in the future.